### DISCUSSION TITLE
How to use drum plugin,  for example SsdSampler, it has 24 output channels

### DISCUSSION THREAD

**Discussion by badd9yang** [DBraun/DawDreamer#186] [Q&A]
for SsdSampler, it has 24output channels. However, only two tracks can be rendered, and only the audio of the first two tracks can be generated.
<img width="485" alt="1701224466846" src="https://github.com/DBraun/DawDreamer/assets/140990330/aa4930a1-b25f-4c11-949d-fa3e19621198">

**Comment by DBraun** [comment]
Have you tried calling `synth.open_editor()` to change some setting? It may also be necessary to try `synth.can_set_bus` and `synth.set_bus` afterward.
